Readme.md

```diff
@@ -3,65 +3,71 @@
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)]()
 [![Gitter](https://img.shields.io/gitter/room/camel-tooling/Lobby.js.svg)](https://gitter.im/camel-tooling/Lobby)
 
-# wsdl2rest for Visual Studio Code
-This preview release of the extension adds wsdl2rest support to [Visual Studio Code](https://code.visualstudio.com/).
+# WSDL to Camel Rest DSL for Visual Studio Code
+This preview release of the extension adds WSDL to REST support to [Visual Studio Code](https://code.visualstudio.com/).
 
-Current functionality includes:
-* Specifying an existing WSDL file in the local file system and generating a Camel Rest DSL + CXF solution around it for REST-style access.
-* Specifying a web-accessible WSDL file and generating a Camel Rest DSL + CXF solution around it for REST-style access.
+ By specifying an existing WSDL file, you can use this extension to generate a Camel Rest DSL + CXF solution for REST-style access. The WSDL file can be located either on your local file system or from an accessible web URL.
 
-
-![wsdl2rest in Command Palette](./images/wsdl2rest-dropdown.png "wsdl2rest in Command Palette")
+Note: For this preview release, the **WSDL to Camel Rest DSL** extension creates Spring Boot and Blueprint configurations. Java configuratons are not supported yet.
 
 ## Contact Us
-If you run into any issues or have suggestions, please file [issues and suggestions on GitHub](https://github.com/camel-tooling/vscode-wsdl2rest/issues).
+If you run into an issue or have a suggestion, you can report it by [creating a new issue on GitHub](https://github.com/camel-tooling/vscode-wsdl2rest/issues).
 
 ## How to install
-The wsdl2rest Extension is available from the [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=camel-tooling.vscode-wsdl2rest).
+You can download this **WSDL to Camel Rest DSL** extension from the Visual Studio Code Marketplace on its own or as part of the [Extension pack for Apache Camel](https://marketplace.visualstudio.com/items?itemName=camel-tooling.apache-camel-extension-pack or as its own [wsdl2rest extension](https://marketplace.visualstudio.com/items?itemName=camel-tooling.vscode-wsdl2rest).
```

Comment:
Missing a ")" after the https://marketplace.visualstudio.com/items?itemName=camel-tooling.apache-camel-extension-pack link